src/components/Book_list.js

```diff
@@ -27,14 +27,16 @@ function BookList() {
               <button type="submit" className="edit-book">Edit</button>
             </div>
           </div>
-          <div>
-            <p className="book-porcentage-number">65%</p>
-            <p className="book-porcentage">COMPLETED</p>
-          </div>
-          <div>
-            <p className="book-character">CURRENT CHAPTER</p>
-            <p className="book-character-number">Chapter 17</p>
-            <button type="submit" className="update-book">UPDATE PROGRESS</button>
+          <div className="box-data">
+            <div className="porcentage-box">
+              <p className="book-porcentage-number">65%</p>
+              <p className="book-porcentage">COMPLETED</p>
+            </div>
+            <div>
+              <p className="book-character">CURRENT CHAPTER</p>
+              <p className="book-character-number">Chapter 17</p>
+              <button type="submit" className="update-book">UPDATE PROGRESS</button>
+            </div>
```

Comment:
- [ ] Please add the correct font family to the elements inside the book card because It's supposed to be `RobotoSlab` instead of `Montserrat` to make it look like the given design.

![image](https://user-images.githubusercontent.com/68796232/185288025-d3b4f0bb-85d0-4407-a237-76ebe6322286.png)

![image](https://user-images.githubusercontent.com/68796232/185287780-c3782a48-6dfa-4eda-92fe-d5594c37803e.png)

### Other than that you have done a great job 💪.